rules/agreements-do-you-use-1-or-2-week-sprints/rule.md

```diff
@@ -1,23 +1,24 @@
 ---
 type: rule
-archivedreason: 
 title: Agreements - Do you have 1 or 2-week Sprint?
-guid: 11cd2bad-7551-4500-a858-2e4743f63aa5
 uri: agreements-do-you-use-1-or-2-week-sprints
-created: 2012-05-01T02:16:06.0000000Z
 authors:
-- title: Ulysses Maclaren
-  url: https://ssw.com.au/people/ulysses-maclaren
+  - title: Ulysses Maclaren
+    url: https://ssw.com.au/people/ulysses-maclaren
 related: []
 redirects: []
-
+created: 2012-05-01T02:16:06.000Z
+archivedreason: null
+guid: 11cd2bad-7551-4500-a858-2e4743f63aa5
 ---
 
 Depending on how much visibility you need on ongoing costs, you will have to decide whether to use 1 or 2 week development iterations.  
 <!--endintro-->
 
 A 1-week Sprint is for small projects (less than 2 months) or if constant visibility into costs is an important factor, as it gives better feedback to the Product Owner.  
-**Note:** This can be at the cost of increased overheads.
+:::greybox
```

Comment:
**nitpick**: The convention throughout Rules is to have a space between the ":::" operator and the fenced block type.

```suggestion
::: greybox
```